odoo/addons/base/models/res_currency.py

```diff
@@ -159,7 +159,7 @@ def _compute_date(self):
     def name_get(self):
         return [(currency.id, tools.ustr(currency.name)) for currency in self]
 
-    def amount_to_text(self, amount):
+    def amount_to_text(self, amount, show_label=True):
```

Comment:
Not sure if it is a good idea to put `show_label` to the `context`. (of course, we have to change its name then)
So you can directly use `o_sec.with_context(show_lable=False).amount_total_words` in the report